src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +10,18 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private static final UserService userService = new UserService();
+
     public boolean login(String email, String password) {
+        if (email == null || password == null) {
+            return false;
+        }
+
+        if (userService.findByEmail(email) != null
+                && password.equals(userService.findByEmail(email).getPassword())) {
+            return true;
+        }
+
         return false;
```

Comment:
- don't call find by email twice, save result to a variable
- don't use true/false literals, return expression itself